src/Codeception/Module/Db.php

```diff
@@ -205,156 +194,271 @@ class Db extends CodeceptionModule implements DbInterface
         'populator' => null,
     ];
 
-    /**
-     * @var bool
-     */
-    protected $populated = false;
-
-    /**
-     * @var \Codeception\Lib\Driver\Db
-     */
-    public $driver;
-
     /**
      * @var array
      */
+    protected $requiredFields = ['dsn', 'user', 'password'];
+    const DEFAULT_DATABASE = 'default';
+
+    public $drivers = [];
+    public $dbhs = [];
+    public $databasesPopulated = [];
+    public $databasesSql = [];
     protected $insertedRows = [];
+    public $currentDatabase = self::DEFAULT_DATABASE;
 
-    /**
-     * @var array
-     */
-    protected $requiredFields = ['dsn', 'user', 'password'];
+    public function getDatabases()
```

Comment:
should not be available inside `$I->`.
So this should be either prefixed with `_` or made protected

Reply:
Fixed